gpa-symbols/src/ReactIcons.tsx

```diff
@@ -0,0 +1,183 @@
+    </svg>
+
+    export const CrossMark: React.FC<IProps> = (props) => <svg xmlns="http://www.w3.org/2000/svg" width={props.Size ?? 24} height={props.Size ?? 24} viewBox="0 0 24 24" fill={props.Color ?? "currentColor"} stroke={props.Color ?? "currentColor"}  strokeWidth="2" >
+        <path d="M 2.2 23.197 L 0.387 21.384 L 10.356 11.415 L 0.387 1.446 L 2.2 -0.367 L 12.169 9.602 L 22.138 -0.367 L 23.951 1.446 L 13.982 11.415 L 23.951 21.384 L 22.138 23.197 L 12.169 13.228 L 2.2 23.197 Z" />
+    </svg>
+
+    export const CircledX: React.FC<IProps> = (props) => <svg xmlns="http://www.w3.org/2000/svg" width={props.Size ?? 24} height={props.Size ?? 24} viewBox="0 0 24 24" fill={props.Color ?? "currentColor"} stroke={props.Color ?? "currentColor"}  strokeWidth="2" strokeLinecap="round" strokeLinejoin="round" className="feather feather-file-text">
+        <circle stroke="rgb(139, 0, 0)" fill={'none'} cx="12" cy="12" r="11"/>
+        <path stroke="rgb(139, 0, 0)" strokeWidth={3} d="M 6 6 L 18 18" />
+        <path stroke="rgb(139, 0, 0)" strokeWidth={3} d="M 18 6 L 6 18" />
```

Comment:
```suggestion
        <circle fill={'none'} cx="12" cy="12" r="11"/>
        <path strokeWidth={3} d="M 6 6 L 18 18" />
        <path strokeWidth={3} d="M 18 6 L 6 18" />
```
Rgb would set the color despite specified prop